app/controllers/browse_controller.rb

```diff
@@ -45,7 +45,7 @@ def self.modified_field
     #   The ordering of the field names is the order of the display
     config.add_facet_field solr_name("resource_type", :facetable), label: "Item Type", limit: 20
     config.add_facet_field solr_name("subject", :facetable), label: "Subject", limit: 20
-    config.add_facet_field solr_name("hasCollection", :symbol), label: "Collection", limit: 20 
+    #config.add_facet_field solr_name("hasCollection", :symbol), label: "Collection", limit: 20 
```

Comment:
I'm not a fan of changes like this.  It's either in or out, commented out code just make it harder to maintain.  Does this mean that the link "Browse collections" from the front page should be changed as well?